### PR TITLE
fix: CodeCov for the repository

### LINKED ISSUE #3537: ReadME: Remove `CodeCov` Badge
Since the `CodeCov` always show `unknown`, Please remove the badge from the `ReadME` file, As it looks odd.

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3537

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ script:
   - ROOT_URL=open-event-frontend ember build -prod
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
+  - 'bash <(curl -s https://codecov.io/bash)'
```
Comment (iamareebjamal) [inline .travis.yml]: what?

Comment (kushthedude) [reply .travis.yml]: The script was not getting executed before. Mow the script is getting execute and also after every build a codecov logo and detail comes on Travis Logs. But why it is not producing any results?

Comment (kushthedude) [reply .travis.yml]: If you can tsll of there is anything more to add, it would be great

Comment (iamareebjamal) [reply .travis.yml]: Why does wrapping it in quotes work?

Comment (kushthedude) [reply .travis.yml]: I dont know, Just read somewhere. Although in worked in CI why its not generating report

Comment (kushthedude) [reply .travis.yml]: @iamareebjamal Any thing on this ?

**Comment by iamareebjamal** [comment]
```
==> Uploading reports
    url: https://codecov.io
    query: branch=development&commit=43b4f6844676fc98c92cd176d791d93853dcbd20&build=10070.1&build_url=&name=&tag=&slug=fossasia%2Fopen-event-frontend&service=travis&flags=&pr=3554&job=597712814
    -> Pinging Codecov
https://codecov.io/upload/v4?package=bash-8a28df4&token=secret&branch=development&commit=43b4f6844676fc98c92cd176d791d93853dcbd20&build=10070.1&build_url=&name=&tag=&slug=fossasia%2Fopen-event-frontend&service=travis&flags=&pr=3554&job=597712814
HTTP 400
Repository not found with provided token.
```

**Comment by kushthedude** [comment]
I think the `token` is only needed for Private Repos, But still, the coverage is unavailable. @iamareebjamal .
Also the report is unavailable at the specifies URL in the script results ?

```sh
Reading reports
4397    + ./coverage/lcov.info bytes=58414
4398==> Appending adjustments
4399    http://docs.codecov.io/docs/fixing-reports
4400    + Found adjustments
4401==> Gzipping contents
4402==> Uploading reports
4403    url: https://codecov.io
4404    query: branch=development&commit=43b4f6844676fc98c92cd176d791d93853dcbd20&build=10070.1&build_url=&name=&tag=&slug=fossasia%2Fopen-event-frontend&service=travis&flags=&pr=3554&job=597712814
4405    -> Pinging Codecov
4406https://codecov.io/upload/v4?package=bash-8a28df4&token=secret&branch=development&commit=43b4f6844676fc98c92cd176d791d93853dcbd20&build=10070.1&build_url=&name=&tag=&slug=fossasia%2Fopen-event-frontend&service=travis&flags=&pr=3554&job=597712814
4407    -> Uploading
4408    -> View reports at https://codecov.io/github/fossasia/open-event-frontend/commit/43b4f6844676fc98c92cd176d791d93853dcbd20
```

**Comment by iamareebjamal** [comment]
Yes, that's why I removed it. Now, it uploads, but not processing. Commented on an issue on codecov support

https://community.codecov.io/t/report-stuck-in-processing-unable-to-find-report-content-in-storage-archive/569/7

**Pull request merged**